src/net/virtio.cc

```diff
@@ -1033,10 +922,3 @@ std::unique_ptr<net::device> create_virtio_net_device(const virtio_options& opts
 }
 
 }
-
-// Locks the shared object in memory and forces on-load function resolution.
-// Needed if the function passed to enable_interrupt() is run at interrupt
-// time.
-// TODO: Instead of doing this, _virtio.enable_interrupt() could take a
-// pollable to wake instead of a function, then this won't be needed.
-asm(".pushsection .note.osv-mlock, \"a\"; .long 0, 0, 0; .popsection");
```

Comment:
This is the only bit I'm 95% sure about instead of 100%. This was being compiled even without HAVE_OSV, adding a note section to the binary and indeed I can find it in the redpanda binary:

```
[ 3] .note.osv-mlock   NOTE             000000000000038c  0000038c
```

However, my very brief research indicates it is OSv-only. Can I get a confirmation?